kolibri/content/api.py

```diff
@@ -0,0 +1,222 @@
+"""
+This module acts as the only interface point between other apps and the database backend for the content.
+It exposes several convenience functions for accessing content
+"""
+from kolibri.content import models as KolibriContent
```

Comment:
Use relative imports when you import from the same package + use isort to sort the imports. An import of something from the same package shouldn't precede imports of external libraries. Built-in Python libs are always imported firstly. But isort handles this :)